modules/ROOT/pages/configure-message-flows.adoc

```diff
@@ -82,15 +83,15 @@ This is the target message type and the Endpoint where you want to route the tra
 . Either search and select an existing message type, or create a new message type. To create a new message type, depending on the message standard, follow instructions in one of these sections.
 
 +
-* xref:document-types.adoc[Message (Document) Types]
+* xref:document-types.adoc[Message Types]
 +
 .  Click *Select* to set up a target endpoint. Either search and select and existing endpoint, or create a new endpoint.
 +
 * To create a new endpoint, depending on the protocol, refer to the following:
 
-* <<SFTP Send Settings>>
+* xref:endpoint-sftp-receive-target.adoc[SFTP Receive Endpoint Settings]
 
-* <<HTTP(S) Protocol Send Settings>>
+* xref:endpoint-https-send.adoc[HTTP and HTTPS Send Configuration]
 
```

Comment:
ditto

Reply:
What follows could be a new topic: Deploy Your Message Flow.
If so, we might want to change the container topic to Create and Deploy a Message Flow.